### ISSUE TITLE
NTR: topsoil 

### ISSUE THREAD

**Issue by kaiiam** [UA-SRC-data/srpdio#5]
We'll need topsoil to describe the USGS soil dataset used in this project. 

From the [wiki topsoil page](https://en.wikipedia.org/wiki/Topsoil) we have the following:

> Topsoil is the upper, outermost layer of soil, usually the top 5–10 inches (13–25 cm). It has the highest concentration of organic matter and microorganisms and is where most of the Earth's biological soil activity occurs. Topsoil is composed of mineral particles, organic matter, water, and air. 

Based on this information I propose the following class:

## Preferred term label

topsoil 

## Textual definition

> Soil which is adjacent to the atmosphere, and is composed of mineral particles, organic matter, water, and air.

## Comment:

> Topsoil typically comprises the top 13–25 cm of the soil profile. Topsoil has high concentrations of organic matter and microorganisms and is where the majority of the Earth's soil  biological activity occurs. 

## Suggested parent term

[soil](http://purl.obolibrary.org/obo/ENVO_00001998)

### Potential axioms:

```
'adjacent to' some 'nitrogen-oxygen planetary atmosphere'
```

```
'composed primarily of' some 
    ('mineral material' or 'soil organic matter' or 'liquid water' or 'air')
```

**Comment by ramonawalls** [comment]
For SRPDIO, I will make the class without axioms, since we don't need to reason on those, and leave the more complex axioms to ENVO, where there is a more lengthy discussion.

**Comment by kaiiam** [comment]
Agreed per the discussion in ENVO, this class is a bit challenging.

**Comment by ramonawalls** [comment]
For some reason, the reasoner is not inferring that top soil is an environmental material, even though it is a subclass of soil, which is an environmental material. Need to look into this.

**Comment by ramonawalls** [comment]
What I meant in the last comment is that the `concentration  of...` hierarchy is not building correctly. The problem seems to be with `concentration of elemental scandium in environmental material` being defined as " 'concentration of' and ('inheres in' some ('chemical entity' and ('part of' some 'liquid water')))", which is wonky, to say the least. Need to check the input files to find the error.

**Comment by kaiiam** [comment]
I'm sure you'll figure it out but if not let me know if you want me to take a look.

**Comment by ramonawalls** [comment]
Closing this issue, and using issue #20, since it is a different problem.

**Comment by ramonawalls** [comment]
The problem with the top soil hierarchy is that top soil is not being added to the ontology. Need to fix SRPDIO_4000007 as part of issue #20
